### PR TITLE
Simplify DNS records section

### PR DESCRIPTION
**Pull request by pijemcolu**
## Description

The suggestions to not use A records was because they weren't static. Now that they are static we can simplify the docs and let the customers use their preferred record sets.

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Umbraco Cloud

## Deadline (if relevant)

n/a

### CODE REVIEW

File: umbraco-cloud/set-up/project-settings/manage-hostnames/README.md [hunk 1]
```diff
@@ -24,19 +24,26 @@ Under **Settings** in the Umbraco Cloud Portal, you'll find **Hostnames**. This
...
+  * `dns.umbraco.io`
+* A records to either or both IPv4 addresses:
+  * `162.159.140.127`
+  * `172.66.0.125`
+* AAAA records to either or both IPv6 addresses (to support IPv6 connectivity):
+  * `2606:4700:7::7d`
+  * `2a06:98c1:58::7d`
 
-If your provider doesn't support setting a CNAME record for your hostname, you can manually set the following records:
+If you are using the [Former A and AAAA records](./#former-a-and-aaaa-records) consider changing them to the new A & AAAA records above as they will not become obsolete.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/manage-hostnames/README.md:40]:
```suggestion
If you're using the [Former A and AAAA records](./#former-a-and-aaaa-records) consider changing them to the A & AAAA records above as they will not become obsolete.
```

**Review by jonat123 — CHANGES_REQUESTED** [review]
@pijemcolu Looks good :D 

I have added a small suggestion.

**Comment by pijemcolu** [comment]
> @pijemcolu Looks good :D
> 
> I have added a small suggestion.

This is now confusing: ```If you are using the [Former A and AAAA records](./#former-a-and-aaaa-records) consider changing them to the new A & AAAA records above as they will not become obsolete.```

We should clear up which records will not become obsolete.

**Comment by jonat123** [comment]
@pijemcolu Maybe the sentence can just be removed, as it is already mentioned that they will be removed in the future in the dropdown 🤔

File: umbraco-cloud/set-up/project-settings/manage-hostnames/README.md (hunk 1 above)
Comment (pijemcolu) [reply umbraco-cloud/set-up/project-settings/manage-hostnames/README.md]: Applied the "shortening" suggestion.

**Comment by pijemcolu** [comment]
> @pijemcolu Maybe the sentence can just be removed, as it is already mentioned that they will be removed in the future in the dropdown 🤔

I've removed the ending, that way it's visible immediately but the unclear ending is gone. Ready to merge now?

**Review by jonat123 — APPROVED** [review]

**Comment by jonat123** [comment]
sure thing, merging the PR :-)

**Pull request merged**